fabric-ingredient-api-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,32 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-ingredient-api-v1",
+  "name": "Fabric Ingredient API (v1)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-ingredient-api-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "mixins": [
+    "fabric-ingredient-api-v1.mixins.json"
+  ],
+  "accessWidener": "fabric-ingredient-api-v1.accesswidener",
+  "depends": {
+    "fabricloader": ">=0.14.9"
```

Comment:
```suggestion
    "fabricloader": ">=0.14.11"
```

Reply:
I just downgraded to `0.14.10` because that's what Fabric API is using at the moment.